Sources/IdentityState.swift

```diff
@@ -94,6 +94,32 @@ class IdentityState {
         createXDMSharedState(identityProperties.toXdmData(), nil)
         return hasBooted
     }
+    
+    /// When the advertising identifier from the `event` is different from the current value, it updates the persisted value and creates
+    /// new shared state and XDM shared state. A consent request event is dispatched when advertising tracking preferences change.
+    /// If privacy is optedout the call is ignored
```

Comment:
This is not accurate anymore as the privacy status is not used in this extension, let's remove this line.